app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,32 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+
+  load_resource only: :show
+
+  # GET /orders
+  def index
+    @orders = current_user.orders
```

Comment:
It possible to replace with `load_and_authorize_resource through: :current_user`